src/main/java/ca/on/oicr/pineryreports/reports/impl/DysReport.java

```diff
@@ -0,0 +1,227 @@
+    }
+
+    public SampleDto getDilution() {
+      return dilution;
+    }
+
+    public RunDto getRun() {
+      return run;
+    }
+
+    public RunDtoPosition getLane() {
+      return lane;
+    }
+    
+    public String getTargetedSequencing() {
+      return targetedSequencing;
+    }
+    
+  }
+  
+  public static final String REPORT_NAME = "dys";
+  
+  public static final String GSLE_USER = "Geospiza";
+
+  private static final List<ColumnDefinition> COLUMNS = Collections.unmodifiableList(Arrays.asList(
+      new ColumnDefinition("External.Identifier"),
+	  new ColumnDefinition("Stock.Alias"),
+	  new ColumnDefinition("Aliquot.Alias"),
+	  new ColumnDefinition("Library.Alias"),
+	  new ColumnDefinition("Run"),
+	  new ColumnDefinition("Barcode"),
+	  new ColumnDefinition("Lane"),
+	  new ColumnDefinition("Targeted.Sequencing"),
+	  new ColumnDefinition("Tissue.Material"),
+	  new ColumnDefinition("Group.ID"),
+	  new ColumnDefinition("Pool.Alias"),
+	  new ColumnDefinition("Num.Dilutions.In.Pool"),
+	  new ColumnDefinition("Pool.Date.Created"),
+	  new ColumnDefinition("Freezer.Box.position")
+  ));
+  
+  private static String DYS = "DYS";
+  Map<String, SampleDto> allSamplesById;
+  List<ReportObject> reportData;
+  
+  @Override
+  public String getReportName() {
+    return REPORT_NAME;
+  }
+
+  @Override
+  public Collection<Option> getOptions() {
+    return Collections.emptySet();
+  }
+  
+  @Override
+  public void processOptions(CommandLine cmd) throws ParseException {
+	    // No options (could be expanded to report for any project, etc)
+  }
+
+  @Override
+  public String getTitle() {
+    return DYS + " Sequencing Report";
+  }
+
+  @Override
+  protected void collectData(PineryClient pinery) throws HttpResponseException {
+    List<SampleDto> samples = pinery.getSample().all();
+    allSamplesById = mapSamplesById(samples);
+    
+    List<ReportObject> rows = new ArrayList<>();
+    List<RunDto> allRuns = pinery.getSequencerRun().all();
+    // get the TargetedSequencing from the RunSampleDtos
+    for (RunDto run : allRuns) {
+      if ("Completed".equals(run.getState()) && run.getPositions() != null) {
+        for (RunDtoPosition pos : run.getPositions()) {
+          if (pos.getSamples() != null) {
+            for (RunDtoSample sam : pos.getSamples()) {
+              SampleDto dilution = allSamplesById.get(sam.getId());
+              if (DYS.equals(dilution.getProjectName())) {
+                rows.add(new ReportObject(dilution, run, pos, getAttribute(sam.getAttributes(), "Targeted Resequencing")));
+              }
+            }
+          }
+        }
+      }
+    }
+    rows.sort(byPoolDate);
+    reportData = rows;
+  }
+  
+  /**
+   * Sort descending by pool date
+   */
+  private final Comparator<ReportObject> byPoolDate = (o1, o2) -> {
+    String o1Created = removeTime(o1.getLane().getPoolCreated());
+    String o2Created = removeTime(o2.getLane().getPoolCreated());
```

Comment:
Probably doesn't matter, but any reason to specifically sort by date and not time?

Reply:
Not especially. Will update.